modules/migration-mtc-release-notes-1-7.adoc

```diff
@@ -23,4 +23,5 @@ This release has the following new features and enhancements:
 This release has the following known issues:
 
 * `MigPlan` custom resource does not display a warning when an AWS gp2 PVC has no available space. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1963927[*BZ#1963927*])
+* Direct and indirect data transfers do not work if the destination storage is a PV dynamically provisioned by the AWS Elastic File System (EFS). This is due to limitations of the AWS EFS Container Storage Interface (CSI) driver.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2085097[*BZ#2085097*])
```

Comment:
1. Consider a small change: ".. if the destination storage is a PV that is dynamically provisioned by the..." 
2. In the future, please add a preview whenever you include a link, so that the reviewer can click it.  If you've checked it already, that's good enough for me. Aside from those items, LGTM. No need for me to review this again.

Reply:
Done